rktl_game/nodes/game_manager

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""Use ball location to determine when a goal is scored.
+License:
+  BSD 3-Clause License
+  Copyright (c) 2010, Autonomous Robotics Club of Purdue (Purdue ARC)
+  All rights reserved.
+"""
+
+
+
+# ROS
+import rospy
+from std_msgs.msg import Bool, Header
+from nav_msgs.msg import Odometry
+from rktl_msgs.msg import Score, MatchStatus
+from std_srvs.srv import Empty, EmptyResponse
+
+class ScoreKeeper():
+    def __init__(self):
+        rospy.init_node('score_keeper')
+
+        self.orange_score = 0
+        self.blue_score = 0
+        self.orange_goal = (rospy.get_param('/field/length', 1) + 0.1) / 2 
+        self.blue_goal = (rospy.get_param('/field/length', 1)  + 0.1) / -2
+        self.enabled = False
+        self.match_status = 0
+        self.game_clock = rospy.get_param('game_length', 90)
```

Comment:
I think we should talk in person about what is going on with the parameters here. They should probably be private (prefixed by `~`), or absolute (prefixed by `/`). Unless if you are namespacing this?